src/test/models/users.model.js

```diff
@@ -6,29 +6,20 @@ import '../../server/models/users.model';
  * Test for mongodb create.
  */
 
-let should = require('should');
+const should = require('should');
 let user = require('mongoose').model('user');
+let u = {
+  name: 'yoda'
+};
 
 describe('Users: models', function () {
   describe('#create()', function () {
-    it('should create a new User', function (done) {
+    it('should create a new User and remove', function (done) {
       // Create a User object to pass to User.create()
-      var u = {
-        name: 'yoda'
-      };
       user.create(u, function (err, createdUser) {
         should.not.exist(err);
         createdUser.name.should.equal('yoda');
-        done();
-      });
-    });
-  
-    it('should delete new User', function (done) {
-      // Create a User object to pass to User.create()
-      var u = {
-        name: 'yoda'
-      };
-      user.find(u).remove(function() {
+        createdUser.remove();
```

Comment:
In method create(), createdUser is exact same instance(collection) that created right after, so I thought find() is not necessary(already verified maybe) and using createdUser is more clear...

But if testDB created, we don't need to consider about removing because it'll always clear DB, right?